content/resources/git-good/index.md

```diff
@@ -199,9 +207,11 @@ The command `git remote add origin https://github.com/<username>/git-good` will
 
 ![](git-remote.png)
 
-Git might prompt you for a username and password to be able to push to your remote repo. The username is the email address you log in with, and for the password you'll need a [Personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).  If you want to save your credentials for later, run `git config --global credential.helper store`, and then the next time you are prompted for an email and access token they will be saved. Be aware that it will be saved as plain text on disk though, so I recommend looking into a more secure way to remotely access GitHub (eg, via SSH).
+Git might prompt you for a username and password to be able to push to your remote repo. The username is the email address you log in with, and for the password you'll need a [Personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
 
-Refresh your GitHub repo page, and you should see your files there.
+**You can read a [full tutorial on how to do this here](@/resources/git-token-authentication/index.md)**
```

Comment:
broken relative link needs to be `@../git-token-authentication/index.md`

Reply:
Fixed broken link - I'm surprised the absolute path doesn't work but I guess this might just be a Zola moment?